packages/bot-web-ui/src/stores/load-modal-store.js

```diff
@@ -119,31 +115,44 @@ export default class LoadModalStore {
     @action.bound
     onActiveIndexChange() {
         if (this.tab_name === tabs_title.TAB_RECENT) {
-            // preview workspace when switch to recent tab
-            if (this.selected_workspace) {
-                this.previewWorkspace(this.selected_workspace_id);
+            if (this.selected_strategy) {
+                this.previewRecentStrategy(this.selected_strategy_id);
+            }
+        } else {
+            // eslint-disable-next-line no-lonely-if
+            if (this.recent_workspace) {
```

Comment:
why didn't you use `else if` instead?

Reply:
IMO it's clearer as we'll know that `this.tab_name === tabs_title.TAB_RECENT` is equal to `false`, and with that in mind we can move on to the next condition, it's more chronological. If I'd use an `else if` we'd have to go back in the code to figure out it's `false`. What do you think?

Reply:
It was just a suggestion, IMO `else if` has the meaning of `this.tab_name === tabs_title.TAB_RECENT` is equal to `false` in itself. However, since both parts have the same result you can do what you think is more readable.